draft-ietf-rats-architecture.md

```diff
@@ -633,7 +633,7 @@ education or previous experience, the employer will contact the respective insti
 former employers to validate the claim. Volunteer organizations often perform police background
 checks on volunteers in order to determine the volunteer's trustworthiness.
 Thus, in this analogy, a prospective volunteer is an Attester, the organization is the Relying Party,
-and a former employer or government agency that issues a report is a Verifier.
+and the agency that issues a report is a Verifier.
```

Comment:
Lines 632-633 used the terms "institutions", "former employers" and "volunteer organizations".
I think "organization" is more generic than "agency" if one wants a single term.
```suggestion
and the organization that issues a report is a Verifier.
```